src/Arcus.Security.Tests.Unit/KeyVault/Doubles/AzureKeyVaultAuthenticatorDummy.cs

```diff
@@ -3,11 +3,11 @@
 using Arcus.Security.Providers.AzureKeyVault.Authentication.Interfaces;
 using Microsoft.Azure.KeyVault;
 
-namespace Arcus.Security.Tests.Unit.KeyVault.Dummies
+namespace Arcus.Security.Tests.Unit.KeyVault.Doubles
```

Comment:
What's a Double? :)

Reply:
A _Test Double_ is a concept that's an umbrella term for all types created to manipulate test input: http://xunitpatterns.com/Test%20Double.html